can-stache-bindings.js

```diff
@@ -755,7 +755,8 @@ require('./converters');
 
 			//!steal-remove-start
 			// user tried to pass something like id="{foo}", so give them a good warning
-			if(ignoreAttribute) {
+			// Something like id="{{foo}}" is ok, though. (not a binding)
+			if(ignoreAttribute && node.value.replace(/{{/g, "").indexOf("{") > -1) {
```

Comment:
can you cache the regexp?  or if there's a way of avoiding a regexp, please do that.  This code is called a lot.  Thanks!